Exercises/exercise_2_btc_transfer_with_c2d2.md

```diff
@@ -0,0 +1,84 @@
+# Exercise 2
+Transfer BTC to Ethereum (as a wrapped asset) and back using `c2d2cli`.
+
+C2D2 is the axelar cross-chain dapp deamon which coordinates the necessary transactions for cross-chain asset transfers.
+The C2D2 CLI automates the steps we performed manually in exercise 1.
+
+## Disclaimer 
+Axelar Network is a work in progress. At no point in time should you transfer any real assets using Axelar. Only use testnet tokens that you're not afraid to lose. Axelar is not responsible for any assets lost, frozen, or unrecoverable in any state or condition. If you find a problem, please submit an issue to this repository following the template. 
+
+## Prerequisites
+- Complete all steps from `README.md`
+
+## What you need
+- Bitcoin testnet faucet to send some test BTC: https://testnet-faucet.mempool.co/
+- Metamask
+- Ethereum Ropsten address (generate via Metamask) 
+
+## Joining the Axelar testnet
+
+Follow the instructions in `README.md` to make sure your node is up to date and you received some test coins to your validator account. 
+
+### Enter the c2d2cli container
```

Comment:
add instructions on how to pull c2d2 image. add version number to `testnet release` in the root.